src/Resource/templates/ResetPasswordController.tpl.php

```diff
@@ -139,8 +139,8 @@ private function processSendingPasswordResetEmail(string $emailFormData, MailerI
         }
 
         $email = (new TemplatedEmail())
-            // @TODO - Set correct email address and name
-            ->from(new Address('noreply@mydomain.com', 'Noreply'))
+            <?php if (null === $from_email || null === $from_email_name) echo '// @TODO - Set correct email address and name'."\n"; ?>
+            ->from(new Address('<?= $from_email ?>', '<?= $from_email_name ?>'))
```

Comment:
Works great on `null`. 

`->from...` gets indented a thousand spaces right if not null